### DISCUSSION TITLE
Not getting my student pack perks.

### DISCUSSION THREAD

**Discussion by otherAdamTHM** [community/community#77720] [GitHub Education]
### Hi everyone,

I've only submitted my application today (and had it approved) and after further research there's many reports of people not getting their perks within the usual 72 hours, as told by @samus-aran. 
I've created this post to let everyone know how long it actually took me and for others to post how long they've been waiting after approval.

![image](https://github.com/community/community/assets/105272515/74a9d4d3-0a7e-4f67-a595-15495f79cd55)
https://github.com/orgs/community/discussions/55559#discussioncomment-5917825

![image](https://github.com/community/community/assets/105272515/02f758c5-9545-44b7-869e-fb2a3bca7f49)

**Comment by Cain-Iriving** [comment]
I am approved in Dec 1, but i don't receive the verification of eductional benefits yet.

**Comment by aramshiva** [comment]
My experience it took around 7 days. It takes a little longer than 72 hours.

**Reply by otherAdamTHM** [reply]
Hey, @aramshiva!

When did you apply, when did you get approved and when did you get your benefits?
More recent submissions (like here https://github.com/orgs/community/discussions/77720#discussioncomment-7750390) have been waiting a little over a week now, maybe it is a busy period for GitHub.

**Reply by aramshiva** [reply]
I believe the delay is them having to contact companies generating new keys/links for you. (Just a guess) It should take 2 weeks max.

**Reply by otherAdamTHM** [reply]
![Screenshot_20231206-133442.png](https://github.com/community/community/assets/105272515/ba03ec39-5c0d-403c-a1ed-d87fe371b6c9)
I have been accepted and have gotten all my perks. Didn't take too long for me

**Reply by aramshiva** [reply]
Glad to hear that!

**Reply by fahim9778** [reply]
@otherAdamTHM 
Congrats! I also applied on 5th Dec and am now in GREEN Approval status (Waiting for Benefits to arrive). BTW I have two questions:

1. How **long** did it take for PURPLE approval? 
2. Can you **use GitHub Copilot for free now?** _(Student Dev can use them for free as per the docs.)_

**Reply by otherAdamTHM** [reply]
Hey, it took me from when I posted this (~4 days)
Yes! I can use GitHub Copilot. I haven't been using this feature but when I was testing it in VS Code it worked perfectly!

**Reply by otherAdamTHM** [reply]
@fahim9778

**Reply by fahim9778** [reply]
@otherAdamTHM 
Wow!! That's optimistic! That means I need to wait a couple of days more. :'(

Also, if you would like to answer, did you apply for re-verify or it's your first time applying for the Student Dev pack? I was previously granted 3 times and never had this waiting period for benefits to arrive. I got them immediately.

**Reply by otherAdamTHM** [reply]
@fahim9778 I attempted to verify during the 2023 summer but it was denied as I was out of the country. Sent in another application with my student ID and it was immediately accepted

**Comment by KanielDatz** [comment]
approved on Nov 26
No benefits yet

**Reply by mdmuhtasimfuadfahim** [reply]
@KanielDatz. It takes almost one week and sometimes more. Please contact [GitHub Support](https://support.github.com/) or email here: support@github.com.
